app/views/api/school_members/index.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+json.array!(@school_members) do |school_member|
+  if school_member.respond_to?(:email) && school_member.email.present?
```

Comment:
❓ trying to follow it through but is `school_member` the direct record from profile? Just wondering if there's another way of determining the role without having to do a join

Reply:
ignore me, will see how easy it is to add a `type` into the profile response so we're not hacking around in different places/apps